collect_app/src/main/res/values/styles.xml

```diff
@@ -10,6 +10,15 @@
         <item name="android:textSize">21sp</item>
     </style>
 
+    <style name="Widget.Collect.TextView.WidgetAnswer" parent="Widget.AppCompat.TextView">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:gravity">center</item>
```

Comment:
I tend to try and keep attributes that position views of styles as it's often useful to be able to see these easily in the layout. Here I'd probably avoid including width, height and gravity.

Reply:
Right! That would make the styles more useable.